src/app/work-item/work-item-list/work-item-list.component.html

```diff
@@ -52,7 +53,7 @@
                 <div class="col-sm-offset-2 col-md-offset-3 col-sm-8 col-md-6">
                   <div class="card-pf card-pf-aggregate-status">
                     <div class="card-pf-title">
-                      <h2 class="pficon pficon-add-circle-o no-wi"></h2>
+                      <h2 class="pficon pficon-add-circle-o no-wi with-cursor-pointer" (click)="typeSelectPanel.openPanel()"></h2>
```

Comment:
this should not be clickable. per the PatternFly Empty State component, there should be a clickable 'Create Work Item' primary button below all of the text.